packages/experimental/Avatar/src/JSAvatar.tsx

```diff
@@ -27,7 +28,7 @@ export const avatarLookup = (layer: string, state: JSAvatarState, userProps: JSA
     layer === userProps['avatarColor'] ||
     (!userProps['avatarColor'] && layer === 'neutral') ||
     layer === avatarSize ||
-    (!userProps['size'] && layer === 'size24') ||
+    layer === 'size' ||
```

Comment:
I'm not sure that you need this line, size is a token but it's not a state or layer... I think the styling will layer correctly without this line

Reply:
fixed